### ISSUE TITLE
White screen when png / jpg resources are used as ImageView drawable

### ISSUE THREAD

**Issue by GraxCode** [siralam/LoopingViewPager#34]
I want to make a looping picture gallery inside of a cardview, this is what the code in activity main looks like:

```xml
   <androidx.cardview.widget.CardView
                android:id="@+id/card_about"
                android:layout_width="match_parent"
                android:layout_height="300dp"
                android:layout_gravity="start"
                android:layout_marginStart="20dp"
                android:layout_marginTop="20dp"
                android:layout_marginEnd="20dp"
                app:cardCornerRadius="12dp"
                app:cardElevation="12dp"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toBottomOf="@+id/title">

            <com.asksira.loopingviewpager.LoopingViewPager
                    android:id="@+id/about_vp"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:clipToPadding="false"
                    app:autoScroll="true"
                    app:isInfinite="true"
                    app:itemAspectRatio="1.33"
                    app:scrollInterval="5000"
                    app:viewpagerAspectRatio="1.78" />
        </androidx.cardview.widget.CardView>
```

My `item_pager.xml`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
        android:layout_width="match_parent"
        android:layout_height="match_parent">

    <ImageView
            android:id="@+id/card_about_img"
            android:layout_width="match_parent"
            android:layout_height="match_parent" />

    <TextView
            android:id="@+id/card_about_txt"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="text" />
</RelativeLayout>
```
My Adapter:
```java
public class AboutAdapter extends LoopingPagerAdapter<String> {

  public AboutAdapter(Context context, ArrayList<String> pictures, boolean isInfinite) {
    super(context, pictures, isInfinite);
  }

  @Override
  protected View inflateView(int viewType, ViewGroup container, int listPosition) {
    return LayoutInflater.from(getContext()).inflate(R.layout.item_pager, container, false);
  }

  @Override
  protected void bindView(View convertView, int listPosition, int viewType) {
    ImageView imageView = convertView.findViewById(R.id.card_about_img);
    // getContext().getResources().getIdentifier(getItem(listPosition), "drawable", getContext().getPackageName());
    imageView.setImageResource(R.drawable.ic_android); // SET IMAGE
    TextView tv = convertView.findViewById(R.id.card_about_txt);
    tv.setText(getItem(listPosition));
  }
}
```
Now when i use `imageView.setImageResource(R.drawable.ic_android);` (with ic_android being a vector asset) it works fine, but when a image asset (png or jpg format) is used, the app doesn't start, you only get a white screen.

**Comment by GraxCode** [comment]
Any ideas?

**Comment by siralam** [comment]
I confirmed there is a problem with the implementation in `onMeasure()`. Since we now have ConstraintLayout to handle aspect ratio for us, I will delete all code inside `onMeasure` and let the parent, ideally a `ConstraintLayout`, to decide the height of `LoopingViewPager`.

**Comment by siralam** [comment]
It will be fixed in v1.4.0.

**Comment by siralam** [comment]
Fixed in v1.4.1.
